devsetup/Makefile

```diff
@@ -174,6 +174,34 @@ edpm_deploy_instance: ## Spin a instance on edpm node
 edpm_play_cleanup: ## Cleanup EDPM openstackansibleee resource
 	-oc delete openstackansibleee deploy-external-dataplane-compute
 
+.PHONY: standalone
+standalone: export EDPM_COMPUTE_NAME=standalone
```

Comment:
The only thing i'm wondering about is the rename here. E.g. `EDPM_COMPUTE_SUFFIX` is used to calculate the IP address, so IIUC we will not be able to deploy a `standalone` and `edpm_compute_0` alongside without further customizations.

Maybe we should customize the IP too if we're customizing the machine name? Or keep using edpm_compute_0? I don't have a strong preference one way or another, but do think it would be nice to get out of potential IP conflict situation between the standalone and edpm_compute_0.

Reply:
I changed it to keep `edpm_compute_0`